api/types/types.go

```diff
@@ -120,6 +120,9 @@ type Container struct {
 	SizeRootFs int               `json:",omitempty"`
 	Labels     map[string]string `json:",omitempty"`
 	Status     string            `json:",omitempty"`
+	HostConfig struct {
+		ContainerHostConfig string `json:"omitempty"`
```

Comment:
Why call it `ContainerHostConfig` instead of `NetworkMode`? It can be other things than `"host"`.